deploy/chart/templates/0000_30_10-olm-operator.deployment.yaml

```diff
@@ -22,10 +22,6 @@ spec:
         - name: olm-operator
           command:
           - /bin/olm
-          {{- if .Values.watchedNamespaces }}
```

Comment:
We don't want to remove this option (it gets used by some of the testing still), we just don't want to set it it some cases